python/py-pysam/Portfile

```diff
@@ -22,12 +21,14 @@ homepage            https://pypi.python.org/pypi/${realname}
 master_sites        pypi:p/${realname}
 distname            ${realname}-${version}
 
-checksums           rmd160  315bdf74c211fad5372c1eb43751eff3609d3d27 \
-                    sha256  04837bf0b1313e57d50076f228463262b9982c410b973eb184c033528f83d523
+checksums           rmd160  f44ed399f80325bb084d254a3d16d97c6bc25f0f \
+                    sha256  658421124c2f3de1b7445e03ca8413df0077f67ea9980abdaab0d1b5f7a8936f \
+                    size    3220356
 
-python.versions     27 34 35 36
+python.versions     27 36 37
```

Comment:
Additionally these ports need to go to `py-graveyard`, but let's not put that into this PR. As soon as graveyard is pulled into the PR, the Travis will start failing and the feedback becomes useless (unless this has been fixed in the meantime). Whoever merges this should also take care of adding the removed ports to `py-graveyard`.

Reply:
I'm going to merge this morning, but I'd like to ask @SoapZA to submit the py-graveyard change.